docs/docs/ref-03-component-specs.md

```diff
@@ -10,6 +10,9 @@ next: tags-and-attributes.html
 
 When creating a component class by invoking `React.createClass()`, you should provide a specification object that contains a `render` method and can optionally contain other lifecycle methods described here.
 
+> Note:
+>
+> It is also possible to use plain JavaScript classes as component classes. These classes can implement most of the same methods, though there are some differences. For more information about these differences, please read our documentation about [ES6 classes](/react/blog/2015/01/27/react-v0.13.0-beta-1.html#es6-classes).
```

Comment:
Can you please link to the documentation (I provided it in the comment in the other PR) and not the blog post?